site/content/en/docs/Prerequisite Knowledge/_index.md

```diff
@@ -36,20 +36,27 @@ The following resources are great for learning these concepts:
 
 ## Kubernetes
 
-Kubernetes builds on top of Docker to run containers at scale, on lots of machines. 
+Kubernetes builds on top of Docker to run containers at scale, on lots of machines.
 If you have yet to learn about Kubernetes, we recommend that you have knowledge in the following
 areas before getting started with Agones:
 
-* Kubernetes as a concept
+* Kubernetes as a concept - you should take the [basics tutorial](https://kubernetes.io/docs/tutorials/kubernetes-basics/)
 * Pods
 * Deployments
 * Services
 * Creating a Deployment with a Service
 
+### Mappings in Agones
+
+Agones extends the Kubernetes API to include a new [custom resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
```

Comment:
We install multiple custom resources (game server, fleet, fleet autoscaler, game server set).

Reply:
I made this plural and added a link to the specs.